gateway/pyproject.toml

```diff
@@ -11,7 +11,7 @@ requires-python = ">=3.10"
 dynamic = ["version"]
 dependencies = [
     # release builds of dstack-gateway depend on a PyPI version of dstack instead
-    "dstack[server] @ https://github.com/dstackai/dstack/archive/refs/heads/master.zip",
+    "dstack[gateway] @ https://github.com/dstackai/dstack/archive/refs/heads/master.zip",
```

Comment:
Seems like a fragile solution since there are no tests that run with `dstack[gateway]`.

Reply:
I will merge for now and think how we can test it.
Although we also don't test CLI or individual backends without `[all]`, so it is not much more fragile than the existing extras.